include/swift/SIL/SILDeclRef.h

```diff
@@ -151,6 +153,8 @@ struct SILDeclRef {
   /// The derivative function identifier.
   AutoDiffDerivativeFunctionIdentifier *derivativeFunctionIdentifier = nullptr;
 
+  GenericSignature specializedSignature;
```

Comment:
Can we stick this in a union with derivativeFunctionIdentifier to save space?

Reply:
Good point.